docs/reference/codelists.md

```diff
@@ -218,6 +378,28 @@ file: ../../codelists/closed/process_type.csv
 ---
 ```
 
+### relationship_type
+
+This codelist is referenced by the following properties:
+
+- `Vulnerability_function/relationship`
+- `Fragility_function/relationship`
+- `Damage_to_loss_function/relationship`
+- `Engineering_demand_function/relationship`
+  \=======
+
+### risk_data_type
+
+This codelist has the following codes:
+
+```{csv-table-no-translate}
+---
+header-rows: 1
+widths: auto
+file: ../../codelists/closed/relationship_type.csv
+---
+```
+
 ### risk_data_type
```

Comment:
No 'This code is referened by' section here?

Reply:
From https://github.com/GFDRR/rdl-standard/pull/120:

> I've temporarily commented out the code in `manage.py` that adds the lists of references to the sub-schema reference and codelist reference as this requires some more work due to the structure of the RDLS schema and I understand that finishing off the schema changes is the priority.

Reinstating the references will involve updating some Python code, but I figured that isn't essential to getting the schema ready for review by the steering committee since the codelist reference page isn't expected to be read end-to-end. Rather, readers are expected to navigate the schema using the schema browser or schema reference pages, which link to the relevant sections of the codelist reference page. Let me know if you think differently.